src/Windowing/Silk.NET.GLFW/GlfwNativeWindow.cs

```diff
@@ -0,0 +1,73 @@
+﻿// This file is part of Silk.NET.
+// 
+// You may modify and distribute Silk.NET under the terms
+// of the MIT license. See the LICENSE file for details.
+
+using System.Runtime.InteropServices;
+using Silk.NET.Core.Contexts;
+
+namespace Silk.NET.GLFW
+{
+    public struct GlfwNativeWindow : INativeWindow
+    {
+        private const int GwlpHInstance = -6;
+    
+        [DllImport("user32", EntryPoint = "GetDC")]
+        private static extern nint Win32GetDC(nint hwnd);
+
+        [DllImport("user32", EntryPoint = "GetWindowLongPtrA")]
+        private static extern nint Win32GetWindowLongPtr(nint hwnd, int index);
```

Comment:
Could really just make user32 / kernel32 bindings at some point 🤔

Reply:
Probably! Will probably be 3.0 though after a BuildTools rewrite - I don't have a lot of faith in it at the moment.